docusaurus/docs/React/release-guides/upgrade-to-v12.mdx

```diff
@@ -246,6 +246,25 @@ Import type alternatives if necessary.
 The `TypingIndicator` component does not render avatars as it used to with legacy stylesheet. Therefore, its prop `Avatar` has been removed.
 
 :::important
-**Action optional**<br/>
+**Action optional**  
 Provide custom `TypingIndicator` through the `Channel` prop.
 :::
+
+## Updated browser target
+
+Version 12 targets browsers that support ES2020. In particular, the code includes `async` functions, optional chaining (`?.`), and nullish coalescing (`??`). These features have been supported by all major desktop and mobile browsers for years, so it made sense for us to raise the baseline.
```

Comment:
On a second thought, maybe it is good if we have a list of browser versions that have full es2020 support.